ibis/backends/duckdb/tests/test_bind.py

```diff
@@ -0,0 +1,9 @@
+def test_binds_are_cast(tpch):
```

Comment:
Probably should just stick this in `test_aggregation.py` in the backends test suite.